README.md

```diff
@@ -61,17 +61,57 @@ if (session.new) {
 const tx = {from: '0xab12...1cd', to: '0x0', nonce: 1, gas: 100000, value: 0, data: '0x0'}
 
 /**
- *  Create transaction
+ *  Send transaction
  */
-const transactionId = await webConnector.createTransaction(tx)
+try {
+  // Submitted Transaction Hash
+  const result = await webConnector.sendTransaction(tx)
+} catch (error) {
+  // Rejected Transaction
+  console.error(error)
+}
 
 /**
- *  Listen to transaction status
+ *  Draft message
  */
-const transactionStatus = await webConnector.listenTransactionStatus(transactionId)
+const msg = 'My email is john@doe.com - 1537836206101'
+
+/**
+ *  Sign message
+ */
+try {
+  // Signed message
+  const result = await webConnector.signMessage(msg)
+} catch (error) {
+  // Rejected signing
+  console.error(error)
+}
+
+/**
+ *  Draft Typed Data
+ */
+const msgParams = [
+  {
+    type: 'string',
+    name: 'Message',
+    value: 'My email is john@doe.com'
+  },
+  {
+    type: 'uint32',
+    name: 'A number',
+    value: '1537836206101'
+  }
+]
 
-if (transactionStatus.success) {
-  const { txHash } = transactionStatus // Get transaction hash
+/**
+ *  Sign Typed Data
+ */
+try {
+  // Signed typed data
+  const result = await webConnector.signTypedData(msgParams)
```

Comment:
Wouldn't that be great if we keep compatible to ETH rpc instead of creating new method for different method.

Example: 

```js
webConnector.sendCallRequest({
   id: 1,
   method: 'eth_sendTransaction',
   params: [{
      from: '0xab12...1cd',
      to: '0x0',
      nonce: 1,
      gas: 100000,
      value: 0,
      data: '0x0'
   }]
})
```

and
 
```js
const msgParams = [
  {
    type: 'string',
    name: 'Message',
    value: 'My email is john@doe.com'
  },
  {
    type: 'uint32',
    name: 'A number',
    value: '1537836206101'
  }
]

webConnector.sendCallRequest({
   id: 1,
   method: 'eth_signedTypedData',
   params: [msgParams]
})
```

That way we don't need to take care of each type of ETH request, standard will be more generic and open.

Reply:
Absolutely! I'm currently changing it on the web3-provider branch as I quickly found it to be easier to match 1:1 the JSON RPC request spec.

I will be updating this PR soon. Thanks for the feedback 🙏